api/Window.json

```diff
@@ -4626,6 +4733,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "relative-multi-screen": {
+          "__compat": {
+            "description": "Relative to the <a href='https://developer.mozilla.org/docs/Web/API/Window_Management_API/Multi-screen_origin'>Multi-screen origin</a>",
+            "support": {
+              "chrome": {
+                "version_added": "100"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "64",
+                "partial_implementation": true,
+                "notes": "Behavior is inconsistent/unreliable."
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
```

Comment:
Seems like Safari's screen.top and window.screenY is always relative to itself, but screen.left and window.screenX are relative to the multi-screen origin... it's hard to understand...